x-pack/plugins/security_solution/server/endpoint/routes/metadata/handlers.ts

```diff
@@ -282,9 +282,53 @@ async function enrichHostMetadata(
       throw e;
     }
   }
+
+  let policyInfo: HostInfo['policy_info'];
+  try {
+    const agent = await metadataRequestContext.endpointAppContextService
+      ?.getAgentService()
+      ?.getAgent(
+        metadataRequestContext.requestHandlerContext.core.savedObjects.client,
+        elasticAgentId
+      );
+    const agentPolicy = await metadataRequestContext.endpointAppContextService
+      .getAgentPolicyService()
+      ?.get(
+        metadataRequestContext.requestHandlerContext.core.savedObjects.client,
+        agent?.policy_id!,
+        true
+      );
+    const endpointPolicy = ((agentPolicy?.package_policies || []) as PackagePolicy[]).find(
+      (x: PackagePolicy) => x.package?.name === 'endpoint'
```

Comment:
nit: why not use a descriptive name than x?

Reply:
was such a tiny function, `x` was just a short ident. like `i,j` in a loop, very little is happening, that the short name matches the small importance. happy to call it `p` or `pol` or `policy` or anything else if you prefer